fdbclient/BackupAgent.actor.h

```diff
@@ -517,14 +517,16 @@ class DatabaseBackupAgent : public BackupAgentBase {
 		    db, [=](Reference<ReadYourWritesTransaction> tr) { return clearPrefix(tr, db, removePrefix); });
 	}
 
+	Future<Version> getCommitVersion(Database dest, Key tagName, ForceAction forceAction, bool doLockDabatabse);
+
+	Future<Void> flushBackup(DatabaseBackupAgent* sourceBackupAgent, Database dest, Key tagName, Version commitVersion);
```

Comment:
This is a function on the backup agent, so we don't need the first parameter.

```suggestion
	Future<Void> flushBackup(Database dest, Key tagName, Version commitVersion);
```

Reply:
OMG I see the point here! I'll fix it now.